.github/workflows/api-build-and-test.yml

```diff
@@ -35,5 +35,19 @@ jobs:
       - name: Lint source files
         run: npm run lint
 
-      - name: Run test
+      - name: Build and start container
+        run: docker-compose -f docker-compose.dev.yml up -d
+
+      - name: Sleep for 1 minute
+        run: sleep 60
+
+      - name: Ping container
+        run: curl localhost:3000/ping
+
+      - name: Create .env file
+        run: |
+          cp sample.env .env
+          cat .env
```

Comment:
```suggestion
```

I would never cat an .env file even if it does not contain anything important.

Reply:
Nice catch, I'd forgotten to remove that after checking if the action runs successfully.